packages/client/src/ActionsAndReducers/playerUi/playerUi_Reducer.ts

```diff
@@ -174,7 +174,7 @@ export const playerUiReducer = (state: PlayerUi = initialState, action: PlayerUi
       return newState
   }
 
-  // console.log(`PlayerUI: ${action.type}`, newState)
+  console.log(`PlayerUI: ${action.type}`, newState)
```

Comment:
This is for your development of this PR, right?

Or do you want it to be permanent?